components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -297,29 +335,50 @@ public function front( $content ) {
 
 			// build Pods object for current item
 			global $post;
-			$pods = pods( $current_post_type, $post->ID );
-
-			if ( $this_pod['single'] && is_singular( $current_post_type ) ) {
-				// load the template
-				$content = $this->load_template( $this_pod['single'], $content, $pods, $this_pod['single_append'] );
-			} //end if
-			// check if we are on an archive of the post type
-			elseif ( $this_pod['archive'] && is_post_type_archive( $current_post_type ) ) {
-				// load the template
-				$content = $this->load_template( $this_pod['archive'], $content, $pods, $this_pod['archive_append'] );
+			$pod_name = $current_post_type;
+			$pod_item = $post->ID;
+			if ( in_the_loop() ) {
+				$pod_name = $post->post_type;
+			} else {
+				// Outside the loop in a taxonomy, we want the term
+				if ( is_tax() ) {
+					global $wp_query;
+					$obj      = $wp_query->get_queried_object();
+					$pod_name = $obj->slug;
+					$pod_item = $obj->term_id;
+				}
+			}
 
-			} elseif ( is_home() && $this_pod['archive'] && $current_post_type === 'post' ) {
-				// if pfat_archive was set and we're in the blog index, try to append template
-				// append the template
-				$content = $this->load_template( $this_pod['archive'], $content, $pods, $this_pod['archive_append'] );
+			$pod_name_and_item = array( $pod_name, $pod_item );
+			/**
+			 * Change which pod and item to run the template against. The
+			 * default pod is the the post type of the post about to be
+			 * displayed, the default item is the post about to be displayed,
+			 * except outside the loop in a taxonomy archive, in which case it
+			 * is the term the archive is for.
+			 *
+			 * @since 2.7.2
```

Comment:
Update version number
```suggestion
			 * @since 2.7.16
```